lib/npm-check-updates.js

```diff
@@ -97,20 +97,54 @@ function createDependencyTable() {
             middle: ''
         }
     });
+    return baseTable;
+}
+
+/**
+ * @param {object} args
+ * @param {object} options
+ * @returns {Promise<object>}
+ */
+async function getOwnerPerDependency(args, options) {
```

Comment:
This would be cleaner taking `from` and `to` and returning `owner`, rather than taking the whole `args` object when it doesn't need other fields.